services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet:
+
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genre__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(genre__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        /,
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> None:
+
+    movie, _ = Movie.objects.create(
+        title=movie_title,
+        description=movie_description
+    )
+    if genres_ids:
+        movie.genre.set(genres_ids)
```

Comment:
The method `set` should be called on `genres` instead of `genre` to match the related name used in the `Movie` model.